client/layout/guided-tours/tours/simple-payments-email-tour.js

```diff
@@ -33,6 +34,46 @@ const handleTargetDisappear = () => {
 	tourFirstStep.style.left = '-9999px';
 };
 
+// IE9+ polyfill for `Element.matches()` used in `DelegatingQuit`
+if ( ! Element.prototype.matches ) {
+	Element.prototype.matches = Element.prototype.msMatchesSelector;
+}
+
+class DelegatingQuit extends Quit {
+	addTargetListener = () => {
+		const { parentTarget } = this.props;
+		const container = targetForSlug( parentTarget );
+
+		if ( container && container.addEventListener ) {
+			container.addEventListener( 'click', this.onClick );
+			container.addEventListener( 'touchstart', this.onClick );
+		}
+	};
+
+	removeTargetListener = () => {
+		const { parentTarget } = this.props;
+		const container = targetForSlug( parentTarget );
+
+		if ( container && container.addEventListener ) {
+			container.removeEventListener( 'click', this.onClick );
+			container.removeEventListener( 'touchstart', this.onClick );
+		}
+	};
+
+	onClick = event => {
+		let eventTarget = event.target;
+		while ( eventTarget !== event.currentTarget ) {
```

Comment:
It would be helpful to add a comment explaining this block of code.

Reply:
Ah, I was hoping that calling the component `DelegatingQuit` would have been enough 😄